eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -14,23 +14,18 @@ parameters:
   TestTimeoutInMinutes: 120
   Matrix:
     Linux_Python35:
-      OSName: 'Linux'
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: '3.5'
     MacOs_Python37:
-      OSName: 'MacOS'
       OSVmImage: 'macOS-10.15'
       PythonVersion: '3.7'
     Windows_Python27:
-      OSName: 'Windows'
       OSVmImage: 'windows-2019'
       PythonVersion: '2.7'
     Linux_PyPy3:
-      OSName: 'Linux'
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: 'pypy3'
     Linux_Python38:
-      OSName: 'Linux'
```

Comment:
While this is good clean-up was it actually needed for the other work?

Reply:
We decided to remove this, and instead create the variable after the verifyagentos script has run. This is the pattern we follow in other places so I updated it here. The variable is only used later to name the test result.